CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ option(SILKIT_BUILD_TESTS "Enable unit and integration tests for the SIL Kit" ON
 option(SILKIT_BUILD_UTILITIES "Build the SIL Kit utility tools" ON)
 option(SILKIT_BUILD_DOCS "Build documentation for the SIL Kit (requires Doxygen and Sphinx)" OFF)
 option(SILKIT_INSTALL_SOURCE "Install and package the source tree" OFF)
+if(SILKIT_INSTALL_SOURCE)
```

Comment:
Jenkins sets SILKIT_INSTALL_SOURCE and SILKIT_BUILD_DOCS based on the buildConfig. I'd suggest we should not link this to be able to create packages without docs.

Reply:
Having `SILKIT_INSTALL_SOURCE=ON` and `SILKIT_BUILD_DOCS=OFF` does not lead to an error during configuration, build, or packaging.

If the `if(...)` is kept, we always force users to have python/sphinx/... setup correctly, even if they are not interested in building the documentation at all (but would like to get a properly installed `SilKit.dll` or `libSilKit.so`).

Reply:
marking this as resolved, as I dropped the "revert of 84445e10ab20d9aaf9ea55147c50c2fed68bff3f" from the patch